ui/src/routes/_layout/organizations/$orgId/create-product.tsx

```diff
@@ -60,7 +60,7 @@ const CreateProductPage = () => {
     onSuccess() {
       toast({
         title: 'Create Product',
-        description: 'New product created successfully.',
+        description: `New product "${form.getValues().name}" created successfully.`,
```

Comment:
I would change all of these to use the return value of the mutations to get the names/etc. from there. I believe the toast will show a wrong name with the current implementation if someone clicks submit and writes something to the input while the submit is processing. The return value should be available to the onSuccess callback.

Reply:
No, I've removed this PR from the merge queue, so you can still fix it.